### DISCUSSION TITLE
Is it possible too use this to use custom ringtone?

### DISCUSSION THREAD

**Discussion by Fotis112** [mcw0/DahuaConsole#19] [Ideas]
Is it possible too use this to use custom ringtone?

something like https://ipcamtalk.com/threads/how-to-use-a-custom-ring-tone-on-a-dahua-vth.51205/

**Comment by mcw0** [comment]
Yes, this script is based on the same referred script at ipcamtalk, you find the function in 'dahua.py' on [Line 1200](https://github.com/mcw0/DahuaConsole/blob/master/dahua.py#L1200)

I'm breaking up the script as it became almost impossible to navigate, and more breaking to come.

**Reply by Fotis112** [reply]
Is there any alternative way to find the Middle number of my VTO? It is not displayed in settings of my VTH...

**Reply by mcw0** [reply]
connect with this console to your VTH and issue this command: config VTOInfo
scroll up to 'Vto00' and you should see it

**Reply by mcw0** [reply]
BTW, if you want output from Dahua internal Console, issue command: setDebug
This has been proven to be needed with VTO/VTH and also my SD.

**Reply by mcw0** [reply]
Please mark it as "answer" if this works out for you, if not, please add some more info.
